tests/test_servable.py

```diff
@@ -1,61 +1,71 @@
-import json
-import time
-
-import pytest
-import sseclient
-
-from hydrosdk.modelversion import LocalModel, ModelVersion
+from hydrosdk.deployment_configuration import DeploymentConfigurationBuilder, DeploymentConfiguration
 from hydrosdk.exceptions import BadRequest
-from hydrosdk.servable import Servable, ServableStatus
-from hydrosdk.cluster import Cluster
+from hydrosdk.modelversion import ModelVersion
 from tests.common_fixtures import *
 from tests.utils import *
 
 
-def test_servable_create(cluster: Cluster, local_model: LocalModel):
+@pytest.fixture(scope="module")
+def mv(cluster: Cluster, local_model: LocalModel):
     mv: ModelVersion = local_model.upload(cluster)
     mv.lock_till_released()
+    return mv
+
+
+@pytest.fixture(scope="module")
+def deployment_configuration(cluster):
+    try:
+        deployment_configuration = DeploymentConfiguration.find(cluster, "servable_dep_config")
+    except BadRequest:
+        deployment_configuration = DeploymentConfigurationBuilder("servable_dep_config", cluster).with_replicas(4).build()
+
+    yield deployment_configuration
+    # DeploymentConfiguration.delete(cluster, deployment_configuration.name)
+
+
+def test_servable_create(cluster, mv: ModelVersion):
     sv: Servable = Servable.create(cluster, mv.name, mv.version)
     assert Servable.find_by_name(cluster, sv.name)
+    # Servable.delete(cluster, sv.name)
```

Comment:
Why deletion was commented?

Reply:
There was (and still I guess) some hard to reproduce the bug when Manager sometimes is unable to find the servable we wish to delete. This triggers randomly ~ for 1 out of 20 `Servable.delete` calls.

I've refactored servable tests and moved Serable creation\deletion to module fixture (as much as I could). This should decrease the frequency of the aforementioned bug.